.travis.yml

```diff
@@ -25,6 +25,7 @@ before_script:
 
 script:
   - ./vendor/bin/parallel-lint src tests
+  - composer run phpstan
```

Comment:
By calling the Composer script, it runs the same configuration as you would run locally.